easybuild/easyconfigs/m/Molden/Molden-5.8-foss-2018a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'MakeCp'
+
+name = 'Molden'
+version = '5.8'
+
+homepage = 'http://www.cmbi.ru.nl/molden/'
+description = """Molden is a package for displaying Molecular Density from the
+ Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical
+ packages Mopac/Ampac"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['ftp://ftp.cmbi.ru.nl/pub/molgraph/molden']
+sources = ['%(namelower)s%(version)s.tar.gz']
+checksums = ['a17bac0901f279e0d89aaf15174875e0695ad7596c5878233946b438d09d93b7']
+
+dependencies = [('X11', '20180131')]
```

Comment:
@RvDijk Can you explain the new `X11` dependency? Is it the new docking feature?

Reply:
@verdurin mainly because it was included in #4082.